_pages/plugins/modularimageanalysis.md

```diff
@@ -0,0 +1,217 @@
+---
+title: ModularImageAnalysis (MIA)
+logo: /media/icons/mia.png
+categories: [Analysis, Automation, Scripting]
+doi: 10.5281/zenodo.1201320
+artifact: io.github.mianalysis:mia
+---
+
+{% include img align="center" name="fig:ModularImageAnalysis screenshot." src="/media/plugins/mia/main-screenshot" %}
+
+## About MIA
+The ModularImageAnalysis (MIA) plugin provides a modular framework for assembling image and object analysis workflows. Detected objects can be transformed, filtered, measured and related. Analysis workflows are batch-enabled by default, allowing easy processing of high-content datasets.
+
+**Compatible:** Designed for "out-of-the-box" compatibility with spatially-calibrated 5D images, yielding measurements in both pixel and physical units.
+
+**Extensible:** Functionality can be extended both internally, via integration with Fiji’s scripting interface, and externally, with Java modules that extend the MIA framework.  Both have full access to all objects and images in the analysis workspace.
+
+**High-throughput:** Workflows are, by default, compatible with batch processing multiple files within a single folder.  Thanks to [BioFormats](/formats/bio-formats), MIA has native support for multi-series image formats such as Leica .lif and Nikon .nd2.
+
+**Modular:** Workflows can be automated from initial image loading through processing, object detection, measurement extraction, visualisation, and data exporting.  MIA includes near 200 modules integrated with key Fiji plugins such as [BioFormats](/formats/bio-formats), [TrackMate](/plugins/trackmate/index) and [Weka Trainable Segmentation](/plugins/tws/index).  For a complete list of modules, please go [here](https://mianalysis.github.io/mia/html/modules/modules.html).
+
+**Reactive:** Module(s) can be turned on/off dynamically in response to factors such as availability of images and objects, user inputs and measurement-based filters.  Switches can also be added to "processing view" for easy workflow control.
+
+MIA is developed in the [Wolfson Bioimaging Facility](http://www.bristol.ac.uk/wolfson-bioimaging/) at the University of Bristol.
+
+## Installation
+### Via the update site
+The latest version of MIA can be installed directly into Fiji via an update site.
+1. Launch the Fiji [Updater](/plugins/updater) from Help > Update...
```

Comment:
It's the ImageJ Updater, not the Fiji Updater. (And really, it's the ImageJ2 Updater, but the component is currently known as the "ImageJ Updater" for historical reasons, so that's how we refer to it for now.)